lib/python/qmk/cli/license_check.py

```diff
@@ -27,7 +28,54 @@ def _simplify_text(input):
     return ' '.join(lines)
 
 
-def _detect_license_from_file_contents(filename, absolute=False):
+def _preformat_license_texts():
+    # Pre-format all the licenses
+    for _, long_licenses in LICENSE_TEXTS:
+        for i in range(len(long_licenses)):
+            long_licenses[i] = _simplify_text(long_licenses[i])
+
+
+def _fixup_relative_paths(inputs):
+    # Fixup inputs if they're not absolute, CLI's working directory is qmk_firmware so relative paths aren't going to be
+    # found unless they're qmk_firmware-local.
+    for i in range(len(inputs)):
+        if not inputs[i].is_absolute():
+            inputs[i] = Path(os.environ.get('ORIG_CWD', os.getcwd())) / inputs[i]
+    return list(sorted(set(inputs)))
```

Comment:
Should be able to reuse some existing functionality with something like 

```py
from qmk.path import normpath
```

```suggestion
    return list(sorted(map(inputs, normpath)))
```

Reply:
Yep, good call. Even simpler implementation added.